tests/beaker_tests/bgp/bgplib.rb

```diff
@@ -360,8 +283,7 @@ def self.create_bgp_manifest_title_pattern1
         ensure                                 => present,
         asn                                    => #{BgpLib::ASN},
       }
-    }
-    EOF"
+    }\nEOF"
```

Comment:
vrf/vrflib does.  I fixed bgp neighbor, already.  But I see that I missed 2 spots in bgplib, so I'll update those.